source/includes/quick-start/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var (
+	uri = os.Getenv("MONGODB_URI")
+)
+
+func main() {
+	ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
```

Comment:
For consistency with the findOne example (https://github.com/mongodb/docs-golang/pull/2/files#diff-9080c13327b8e149406ba2a276b31c02c0e664eb9db2a5a6c8c348f2a9a037ffR16), it may be better to use `context.TODO()` as a placeholder context.